pkg/plugin/plugin_init.go

```diff
@@ -55,26 +53,35 @@ func InitPlugin(pluginName string, newEventHandler EventHandlerCreator, newServe
 
 	flag.Parse()
 
-	log := configureLogrus(pluginName)
+	logger := log.ConfigureLogrus(pluginName)
+	sentryDsn, err := utils.LoadSecret(*sentryDsnSecretFile)
+	if err != nil {
+		logger.WithError(err).Errorf("unable to load sentry dsn from %q. No sentry integration enabled", sentryDsnSecretFile)
+	}
+
+	log.AddSentryHook(logger, log.NewSentryConfiguration(string(sentryDsn), map[string]string{
+		"plugin": pluginName,
+		"environment": *environment,
+	}, *sentryTimeout))
```

Comment:
This whole part could be moved to separated function located either in this file or in the `log` package, no make the `InitPlugin` function shorter.

Reply:
I thought `log.AddSentryHook` is the function, the rest is just constructing proper config based on init of the plugin (flags).

Reply:
Eventually I did that as e287906... and ... it had a bug :P